Added let vs var differences

sai-karthik-p opened the pull request:
#9 Added Javascript Program for 'let vs var differences' with comments and explanation

sai-karthik-p committed:
Added let vs var differences

Added four let vs var differences with comments and explaination

snehakoshti commented on let vs var.js:
```diff
@@ -0,0 +1,69 @@
+
+// var: creates a property on global window object when a var variable is created in global scope
+let fruit = "Orange";
+console.log(window.fruit); //Orange
```
here datatype should be var.
you have given result according to var data-type

snehakoshti commented on let vs var.js:
```diff
@@ -0,0 +1,69 @@
+
+// var: variable declaration is hoisted, and set to 'undefined'
+console.log(tree); //undefined
+let tree = "oak";
```
again here also datatype should be var

snehakoshti commented on let vs var.js:
```diff
@@ -0,0 +1,69 @@
+    var varFruit = "apple";
+    console.log(varFruit); // apple // (enclosing function scope)
+}
+console.log(varFruit); //ReferenceError: varFruit is not defined
```
output in comment is wrong.
both place we will get "apple" output

snehakoshti reviewed (commented):
please check comment

sai-karthik-p committed:
Resolved PR comments

1. Resolved PR comments
2. Added some code for `enclosing function scope` in var which was missing earlier

sai-karthik-p commented:
@snehakoshti Thanks for pointing out.

1. I've resolved PR comments
2. And also added some code for `enclosing function scope` in var which was missing earlier

snehakoshti reviewed (approved):
Approved

snehakoshti commented:
PR merged
